lib/token_mods.py

```diff
@@ -0,0 +1,83 @@
+import os
+import os.path
+import shutil
+from typing import List, Set
+
+
+def get_job_scopes(
+    tokenfile: str, need_modify: List[str], need_scopes: List[str]
+) -> List[str]:
```

Comment:
Just like the other functions in this module, could you please add a quick comment/docstring saying what this function does?  From what I gather, it's to add modify scopes to a token, if they're allowed, or any other scope that's needed.  Is that right?